src/components/mods/modsTable.tsx

```diff
@@ -121,7 +167,13 @@ export const ModsTable = ({ qualities, difficulties, modsWithInfo, isLoading }:
     const [mapCountRange, setMapCountRange] = useState<[number | undefined, number | undefined]>([undefined, undefined]);     //[min, max]
     const [selectedModTypes, setSelectedModTypes] = useState<ModType[]>([]);
     const [selectedQualities, setSelectedQualities] = useState<string[]>([]);
-    const [selectedParentDifficulties, setSelectedParentDifficulties] = useState<string[]>([]);
+    const [selectedChildDifficulties, setSelectedChildDifficulties] = useState<string[]>([]);
+
+    // Reset tab index and selected child difficulties if the difficulties change.
+    useEffect(() => {
```

Comment:
This effect runs when `difficulties` loads for the first time as well as when the `difficulties` returned by the server changes (this only happens when we change the difficulties in the database).